test/features.js

```diff
@@ -0,0 +1,79 @@
+import {setup, setupWithDoc, teardown, updateDocument} from './utils';
```

Comment:
Part 2; This file is incomplete, the actual tests still need to be ported. For an example, see pouchdb-validation and/or pouchdb-auth. The http.js file which is finished might also help.